docs/resources/diagrams.md

```diff
@@ -25,6 +25,10 @@ Additional symbols for Microsoft Office and related technologies are available i
 
 A version of the diagrams used in the [Reference Architectures](../reference-architectures/index.md) is available for [download in Visio format](https://aka.ms/arch-diagrams).
 
+## Solution Architecture icons
+
+Microsoft publishes [solution architectures and accompanying diagrams](https://azure.microsoft.com/en-us/solutions/architecture/). The diagrams are in downloadable in SVG format. SVG can be opened and then modified by many tools, including Visio and PowerPoint. If you upgroup the diagram, you can get at the individual icons.  To download a diagram, right-click on it and choose "Save As". You should see an option to save as SVG. If you do not see this option, try again with a different browser. 
```

Comment:
We need to remove the `/en-us/` from the URL (and  test it because some times it won't work 🤕 ).

Reply:
Changed made.